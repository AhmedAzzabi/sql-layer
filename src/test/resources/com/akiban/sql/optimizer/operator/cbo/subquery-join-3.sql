SELECT COUNT(*) FROM customers
 INNER JOIN (SELECT i1.iid + 1 AS v0, i2.iid + 1 AS v1 FROM items i1 INNER JOIN items i2 ON i1.quan = i2.quan) AS v ON v.v0 = cid
  LEFT JOIN orders ON v.v1 = oid