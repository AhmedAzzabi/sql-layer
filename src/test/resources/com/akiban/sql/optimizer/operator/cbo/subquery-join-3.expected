PhysicalSelect[_SQL_COL_1:INTEGER NOT NULL]
  project([Field(0)])
    Count_Default(Index(test.orders.PRIMARY[IndexColumn(oid)]))
      Map_NestedLoops
        Map_NestedLoops
          Filter_Default([test.items])
            GroupScan_Default(full scan on _akiban_customers)
          project([Arith(+ -> Bound(1,Field(0)), Literal(1) -> LONG), Arith(+ -> Field(0), Literal(1) -> LONG)])
            Filter_Default([test.items])
              Select_HKeyOrdered(test.items, Field(3) == Bound(1,Field(3)))
                GroupScan_Default(full scan on _akiban_customers)
        Map_NestedLoops
          IndexScan_Default(Index(test.customers.PRIMARY[IndexColumn(cid)]) (>=UnboundExpressions[Bound(0,Field(0))],<=UnboundExpressions[Bound(0,Field(0))]))
          IfEmpty_Default(Literal(NULL), Literal(NULL), KEEP_INPUT)
            IndexScan_Default(Index(test.orders.PRIMARY[IndexColumn(oid)]) (>=UnboundExpressions[Bound(0,Field(1))],<=UnboundExpressions[Bound(0,Field(1))]))